handbook/engineering/environments.md

```diff
@@ -0,0 +1,25 @@
+# Environment
+
+This page describes our different cloud environments.
+
+## Google Cloud
+
+We utilize multiple Google Cloud projects and folders to organize our workloads and manage access control.
+
+
+### Root Projects
+
+- **Sourcegraph**: Services for sourcegraph.com.
+- **sourcegraph-code-intel**: Services for Code Intel code execution.
```

Comment:
I think this should document why these exist. For example, we created this for isolation purposes.

Reply:
I can update the main description to explain that as its a general attribute more than a per-project one.

Reply:
I believe this should be covered by "We utilize multiple Google Cloud projects and folders to organize our workloads and manage access control." in the GCP section.

Reply:
@nicksnyder Updated, let me know your thoughts.